cluster/sbin/snappy-status-all.sh

```diff
@@ -20,6 +20,11 @@
 # Start all snappy daemons - locator, lead and server on the nodes specified in the
 # conf/locators, conf/leads and conf/servers files repsectively
 
+if [ "$#" -gt 0 ]; then
+  echo "Invalid argument::This shell script doesn't require any agrument."
```

Comment:
Usage: Run this script without any arguments.

Reply:
Resolved